src/utils/test-utils.tsx

```diff
@@ -53,27 +70,26 @@ const customRender = async (
 ) => {
     const { route, username } = options;
 
-    goTo(route, 'Test Page');
-
     const view = rtlRender(
         <AppContent>
-            <MockClientProvider>
-                <MockUserProvider username={username}>
-                    <ThemeProvider>
-                        <AppRouter>{ui}</AppRouter>
-                    </ThemeProvider>
-                </MockUserProvider>
-            </MockClientProvider>
+            <MockProviders username={username}>
+                <ThemeProvider>
+                    <AppRouter>{ui}</AppRouter>
+                </ThemeProvider>
+            </MockProviders>
         </AppContent>,
         {
             ...options,
         }
     );
 
-    await waitForLoading();
+    goTo(route, 'Test Page');
```

Comment:
We wait for stuff in each test - so no need for waiting to load.

Putting the route after the router is created.